metasrc/tex/lsstdoc.py

```diff
@@ -185,6 +190,41 @@ def is_draft(self):
             return False
 
     @property
+    def revision_datetime(self):
+        """Current revision date of the document (`datetime.datetime`).
+
+        The `revision_datetime_source` describes how the revision date
+        is computed.
+        """
+        if not hasattr(self, '_datetime'):
+            self._parse_revision_date()
+
+        return self._datetime
+
+    @property
+    def revision_datetime_source(self):
+        r"""Data source for the `revision_datetime` attribute (`str`).
+
+        Possible string values are:
+
+        - ``'now'``: The current date, when ``\today`` is the content of
+          the document's ``\date`` command. ``YYYY-MM-DD`` dates are converted
+          to UTC datetimes by assuming the document is released at the
+          beginning of the day in the ``US/Pacific`` timezone.
+
+        - ``'git'``: The latest Git commit's timestamp that affected document
+          content. Content is considered any file with a ``tex``, ``bib``,
+          ``pdf``, ``jpg``, or ``png`` extension.
+
+        - ``'tex'``: The document revision date is defined in the ``\date``
```

Comment:
I am considering ignoring the `\date` command if draft mode has been enabled. I must remember that this will now require changes here as well.